content/en/security/application_security/threats/attackers.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Attackers
+kind: documentation
+---
+
+# Understanding Attackers
+
+ASM classifies IP addresses as
+Benign
+Suspicious
+Flagged
+
+ASM additionally clusters IPs with similar attributes together.
+
+Benign: Traffic from APM that has not recently sent attack traffic
```

Comment:
Should we reformulate like this to be clearer?
```
Benign: IPs that haven't sent any significant amont of attack attempts
```

Reply:
Updated based on latest draft, which removes this.